langchain4j/src/main/java/dev/langchain4j/service/AiServiceTokenStreamParameters.java

```diff
@@ -234,14 +235,8 @@ public Builder context(AiServiceContext context) {
             return this;
         }
 
-        /**
-         * Sets the memory ID.
-         *
-         * @param memoryId the memory ID
-         * @return this builder
-         */
-        public Builder memoryId(Object memoryId) {
```

Comment:
This is a more general question: do we want to remove the possibility to set the `memoryId` alone here, instead of deprecating the method as you did somewhere else? And even about deprecation, I'm not sure of it: why should we deprecate or even remove a method that is convenient for users who don't need to deal with a complete `InvocationContext` (provided that we are always able to deal internally with it in a completely unambiguous way)?